src/components/TimeseriesChart.js

```diff
@@ -848,7 +848,10 @@ function mapStateToProps(state) {
       } else {
         for (let longUuid in state.rasterEvents) {
           shortIntersectionUuid = longUuid.slice(0, 7);
-          if (shortIntersectionUuid === intersectionUuid) {
+          if (
```

Comment:
I really hate that "short raster UUIDs" exist and also wish that a check like this were abstracted away into a function. a TimeseriesChart component shouldn't have to know about such details.